docs/guide/src/dev/parameter_setup.md

```diff
@@ -33,10 +33,10 @@ Then edit `penumbra-proof-params` to reference the new parameters created in
 
 ## Circuit Benchmarks
 
-We have benchmarks for all proofs in the `penumbra-proof-params` crate. You can run them via:
+We have benchmarks for all proofs in the `penumbra-bench` crate. You can run them via:
 
 ```shell
-cd crates/crypto/proof-params
+cd crates/bench
```

Comment:
Nit: do we really need to `cd`? Running `cargo bench` at the top level repo root seems to work for me.

Reply:
we don't, removed